src/ui/components/SecondaryToolbox/index.tsx

```diff
@@ -160,7 +166,7 @@ function SecondaryToolbox() {
         </div>
       </header>
       <Redacted className="secondary-toolbox-content bg-chrome text-xs">
-        {recordingCapabilities.supportsNetworkRequests && (
+        {(chromiumNetMonitorEnabled || recordingCapabilities.supportsNetworkRequests) && (
```

Comment:
Hmm, this reads a bit confusing to me. So the `chromiumNetMonitorEnabled` is an experimental thing? Is it tied to the recording at all?

It seems like this might be less confusing if `recordingCapabilities.supportsNetworkRequests` just took this into consideration– but maybe that doesn't make sense for some reason?

Reply:
That was my initial idea, but I wasn't comfortable with mixing user settings with the protocol threadfront...

Reply:
That makes sense.

It's a little weird to see "not supported" but then we do it anyway– but it's probably fine 😁